packages/midi-input/src/MidiInput.ts

```diff
@@ -1,18 +1,21 @@
 import {
   getNextEnumValue,
-  getParamConfig,
   HedronEngine,
+  InputOptionNodesConfig,
   IPlugin,
-  NodeTypes,
   NodeValue,
-  SketchConfigParamEnum,
+  handleEachInput,
 } from '@hedron/engine'
 import { MidiManager, MidiMessageType } from '@hedron/midi-manager'
 
-export interface MidiInputOptions {
-  channel: number
-  note: number
-  type: MidiMessageType
+const noteLetters = ['C', 'C#', 'D', 'D#', 'E', 'F', 'F#', 'G', 'G#', 'A', 'A#', 'B']
+
+const midiNotes: string[] = new Array(128)
+
+for (let i = 0; i < 128; i++) {
+  const letter = noteLetters[i % noteLetters.length]
+  const octave = Math.floor(i / noteLetters.length) - 1
+  midiNotes[i] = `${i} - ${letter} (${octave})`
```

Comment:
@cale-bradbury it would be great if you could test the MIDI options and let me know if you're okay with the note names like this? It's the same as the old hedron